action.yml

```diff
@@ -57,6 +57,9 @@ inputs:
     required: false
     type: boolean
     default: 'true'
+  release-artifact-build-folder:
```

Comment:
I'd keep this more generic and call it `artifact-path`. 

Also, since we're introducing this new feature and although we don't need it at the current time, I'd consider making it generic enough to support either folders or files. Then, in our code we can figure out if it's a folder or a file and in the first case zip the folder, in the second send the file as-is (or zip the file perhaps?)